types/react-fa/index.d.ts

```diff
@@ -37,7 +37,6 @@ export const Icon: ComponentClass<IconProps>;
 
 export interface IconStackProps extends SizeOverrideHTMLProps<IconStack> {
     size?: IconSize;
-    children?: IconProps[];
```

Comment:
this change is to protect against incoming `@types/react` changes that result in these type definitions being broken. I looked through `react-bootstrap`'s typings and noticed they also do not type children, even when it is required. I am mostly interested in fixing this before other people pull in `@types/react@15.0.24`